### PR TITLE
Hydra controller updates and test code

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

File: libraries/input-plugins/src/input-plugins/SixenseManager.h [hunk 1]
```diff
@@ -88,9 +71,9 @@ public slots:
     void setSixenseFilter(bool filter);
 
 private:    
-    void handleButtonEvent(unsigned int buttons, int index);
-    void handleAxisEvent(float x, float y, float trigger, int index);
-    void handlePoseEvent(glm::vec3 position, glm::quat rotation, int index);
+    void handleButtonEvent(unsigned int buttons, bool left);
```
Comment (samcake) [inline libraries/input-plugins/src/input-plugins/SixenseManager.h]: JUst curious,
In the previous version it seems that left was #0 
Using a bool called "left" it sounds like left is now #1 no ?

**Comment by hifi-gustavo** [comment]
Build console output for commit: abb3ee7d0d4154a271aab3266e92ef7c15ffaa0f available at: https://gist.github.com/e6c7b411a31de2bc07dd<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6101.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6101.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5d512d22fbae21f7e1e799b823cfea5de5239916 available at: https://gist.github.com/e6c7b411a31de2bc07dd<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6101.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6101.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: db0fa6b8edfdd9571f68b21a3a176d0f76627ad7 available at: https://gist.github.com/e6c7b411a31de2bc07dd<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6101.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6101.dmg

**Pull request merged**